wordpress-mobile/gutenberg-mobile
# Ignore i18n cache files

As a follow-up of https://github.com/wordpress-mobile/gutenberg-mobile/pull/4323, the i18n cache files are no longer required to be versioned hence, we can ignore them as they are fetched on the fly when generating the JS bundle.

To test:
- Run command: `npm run bundle`.
- Observe that no files under the folder `src/i18n-cache` are modified.
- Expect bundle files to be updated in `bundle/ios` folder.

PR submission checklist:
